websites/T/Tanks.gg/presence.ts

```diff
@@ -37,9 +37,7 @@ presence.on("UpdateData", async () => {
 		];
 	} else if (document.location.pathname.startsWith("/techtree/")) {
 		presenceData.details = "Viewing the tech tree";
-		presenceData.state = `${
-			document.querySelector(".current").innerHTML
-		}`;
+		presenceData.state = `${document.querySelector(".current").innerHTML}`;
```

Comment:
Since you're not passing anything in the string other than the query that return a string, this is useless.
Also, prefer to use textContent since innerHTML is less clear when you read and also less fast than textContent.
```suggestion
		presenceData.state = document.querySelector(".current").textContent;
```